saleor/product/models.py

```diff
@@ -313,6 +313,9 @@ def is_available_for_purchase(self):
             and datetime.date.today() >= self.available_for_purchase
         )
 
+    def get_default_variant(self):
```

Comment:
We should avoid this kind of utils. Each usage of this function generate 1 database query.

Reply:
I would also avoid any model methods and instead created an optimal way of returning default variants in the resolver in the `Product` type.

Reply:
Another option is to have this as a utility function somewhere in `product/utils` but it should be prepared to work with dataloader later so accept list of variants as an argument and instead of doing filter it could just iterate over them and return the first variant where `default=True`.